topic-operator/src/test/java/io/strimzi/operator/topic/v2/TopicControllerIT.java

```diff
@@ -96,7 +97,7 @@ class TopicControllerIT {
 
     private static final Logger LOGGER = LogManager.getLogger(TopicControllerIT.class);
     public static final Map<String, String> SELECTOR = Map.of("foo", "FOO", "bar", "BAR");
-    private static final String NAMESPACE = "ns";
+    private static final String NAMESPACE = "test-" + UUID.randomUUID();
```

Comment:
Why was this necessary?

Reply:
This is just to minimize the risk of using an existing test namespace on the test cluster. Additionally, in case of test termination, you can run the suite again and delete the old namespace in parallel.

Reply:
The problem is you can end up not knowing what namespace it's using. It's annoying if you find you've got lots of `test-...` namespaces in your cluster and you don't know which was the one used for the test that just failed. But :man_shrugging: if you really want this

Reply:
> It's annoying if you find you've got lots of test-... namespaces in your cluster and you don't know which was the one used

Yes, haven't thought about that. Will change to "uto-test".